technical-overview.md

```diff
@@ -55,17 +55,16 @@ principles.
 
 ### Everything's a Widget
 
-Widgets are the basic building blocks of every Flutter app. Each widget is an
+Widgets are the basic building blocks of a Flutter app's UI. Each widget is an
 immutable declaration of part of the user interface.  Unlike other frameworks that
-separate views, controllers, layouts, and other properties, Flutter has a
+separate views, view-controllers, layouts, and other properties, Flutter has a
 consistent, unified object model: the widget.
```

Comment:
we have controllers (separate from widgets) too. And our widgets have properties.